river/tree/splitter/ebst_splitter.py

```diff
@@ -280,3 +243,56 @@ def _remove_bad_split_nodes(self, current_node, parent=None, is_left_child=True)
                 return True
 
         return False
+
+
+class Node:
```

Comment:
In excess of caution, should we name this as `EBSTNode`?

Reply:
I agree with you, it is better to make things clear. Renamed in commit [3b4fed1](https://github.com/online-ml/river/pull/488/commits/3b4fed13fee51b7f77a3b5226ac24f27894f44ff).